src/test/java/seedu/address/storage/StorageManagerTest.java

```diff
@@ -40,6 +43,9 @@ public void prefsReadSave() throws Exception {
         UserPrefs original = new UserPrefs();
         original.setGuiSettings(new GuiSettings(300, 600, 4, 6));
         storageManager.saveUserPrefs(original);
+
+        assert storageManager.readUserPrefs().isPresent();
```

Comment:
The assert statement here makes sense in this context. LGTM.

You can also consider: 

Since you are saving and retrieving GUI settings within the UserPrefs, you can perhaps also add assertions to verify each GUI setting separately.